docs/self-managed/platform-deployment/helm-kubernetes/platforms/redhat-openshift.md

```diff
@@ -281,34 +281,49 @@ If you should decide to use the Red Hat endorsed [NGINX Ingress Controller](http
 
 ### Prerequisite
 
-As the Zeebe Gateway uses `gRPC` (which relies on `HTTP/2`), this [has to be enabled](https://docs.openshift.com/container-platform/4.11/networking/ingress-operator.html#nw-http2-haproxy_configuring-ingress).
+As the Zeebe Gateway also uses `gRPC` (which relies on `HTTP/2`), this [has to be enabled](https://docs.openshift.com/container-platform/4.11/networking/ingress-operator.html#nw-http2-haproxy_configuring-ingress).
 
 ### Required steps
 
 1. Provide [TLS secrets](https://kubernetes.io/docs/concepts/configuration/secret/#tls-secrets) for the Zeebe Gateway, the [Cert Manager](https://docs.openshift.com/container-platform/4.11/security/cert_manager_operator/index.html) might be helpful here:
 
-- One issued to the Zeebe Gateway Service Name. This must use the [pkcs8 syntax](https://www.openssl.org/docs/man3.1/man1/openssl-pkcs8.html) as Zeebe only supports this, referenced as **Service Certificate Secret** or `<SERVICE_CERTIFICATE_SECRET_NAME>`. For more details, review the [OpenShift documentation](https://docs.openshift.com/container-platform/4.11/networking/routes/secured-routes.html#nw-ingress-creating-a-reencrypt-route-with-a-custom-certificate_secured-routes).
+- One issued to the Zeebe Gateway Service Name. This must use the [pkcs8 syntax](https://www.openssl.org/docs/man3.1/man1/openssl-pkcs8.html) or [pkcs1 syntax](https://en.wikipedia.org/wiki/PKCS_1) as Zeebe only supports these, referenced as **Service Certificate Secret** or `<SERVICE_CERTIFICATE_SECRET_NAME>`. For more details, review the [OpenShift documentation](https://docs.openshift.com/container-platform/4.11/networking/routes/secured-routes.html#nw-ingress-creating-a-reencrypt-route-with-a-custom-certificate_secured-routes).
 - One that is used on the exposed route, referenced as **External URL Certificate Secret** or `<EXTERNAL_URL_CERTIFICATE_SECRET_NAME>`.
 
 1. Configure your Zeebe Gateway Ingress to create a [re-encrypt route](https://docs.openshift.com/container-platform/4.11/networking/routes/route-configuration.html#nw-ingress-creating-a-route-via-an-ingress_route-configuration):
 
 ```yaml
-zeebe-gateway:
+zeebeGateway:
   ingress:
-    annotations:
-      route.openshift.io/termination: reencrypt
-      route.openshift.io/destination-ca-certificate-secret: <SERVICE_CERTIFICATE_SECRET_NAME>
-    className: openshift-default
-    tls:
-      enabled: true
-      secretName: <EXTERNAL_URL_CERTIFICATE_SECRET_NAME>
+    grpc:
+      annotations:
+        route.openshift.io/termination: reencrypt
+        route.openshift.io/destination-ca-certificate-secret: <SERVICE_CERTIFICATE_SECRET_NAME>
+      className: openshift-default
+      tls:
+        enabled: true
+        secretName: <EXTERNAL_URL_CERTIFICATE_SECRET_NAME>
+    rest:
```

Comment:
:pray: Not sure here either, I'm not familiar with the OpenShift routes. Is the idea that for plain HTTP an ingress wouldn't be required (as none of the other web apps have a section here)?

Reply:
This section is fine with the changes you made.  the trickiest part of openshift routes was dealing with GRPC since GRPC isn't supported in openshift routes.  Creating a route against the REST endpoint should be easy in comparison.

Reply:
I can confirm. With the shift to REST, Openshift will become much easier to handle (given job streaming will only happen inside the cluster, this section can be removed completely as obsolete)

Reply:
So I would revert these changes.

Reply:
Please revert these changes and only adjust to the new zeebeGateway.ingress configuration.

Reply:
> Please revert these changes and only adjust to the new zeebeGateway.ingress configuration.

Can you clarify what you mean? So we only keep the `zeebeGateway.ingress.grpc` part? With 8.5, you're still required to use gRPC.

Reply:
Yes, exactly. I would not mention the rest part to prevent confusion.